.vscode/settings.json

```diff
@@ -9,9 +9,9 @@
   "typescript.format.enable": true,
   "json.format.enable": true,
   "html.format.enable": true,
-  "editor.formatOnSave": false,
+  "editor.formatOnSave": true,
```

Comment:
We should have a gitignore rule for this, but for now, don't include these changes to vscode settings